src/main/java/seedu/avo/tasks/Task.java

```diff
@@ -0,0 +1,37 @@
+package seedu.avo.tasks;
+
+import java.time.LocalDate;
+
+public abstract class Task {
+     private boolean isCompleted;
+     private final String name;
+     public Task(String name) {
+          this.name = name;
+          isCompleted = false;
+     }
+     public void complete() {
+          isCompleted  = true;
+     }
+     public void unComplete() {
```

Comment:
**Suggestion**: The `unComplete` method name seems a bit unconventional. Could you please consider a more precise or commonly used term to replace it?